pt/esbocos/originais/git-ferramenta-metodologica-projetos-historia-1.md

```diff
@@ -304,33 +304,33 @@ No ramo main
 No commits yet
 
 Arquivos não monitorados:
-  (utilize "git add <arquivo>..." para incluir o que será submetido)
+  (utilize "git add <ficheiro>..." para incluir o que será submetido)
 	README.md
 
-nada adicionado ao envio mas arquivos não registrados estão presentes (use "git add" to registrar)
+nada adicionado ao envio mas arquivos não registados estão presentes (use "git add" to registar)
```

Comment:
Tendo em conta o glossário, "Arquivos" poderá ser alterado para "Ficheiros"? Ou no código aparece mesmo como "Arquivos"? Será pela utilização de PT do BR? Se for podemos deixar uma nota no início da lição.

Reply:
Neste caso, assim como no anterior, devemos manter arquivo, registrados e registrar, pois assim é a tradução do programa para o pt_BR.

Reply:
Alterado. Obrigada!